R/tab_create_modify.R

```diff
@@ -247,7 +254,30 @@ tab_spanner_delim <- function(data,
 
     split_colnames <- strsplit(colnames_with_delim, delim, fixed = TRUE)
 
-    spanners <- vapply(split_colnames, `[[`, character(1), 1)
+    if (split == "first") {
```

Comment:
For next time: we can do this just by varying the pattern

Reply:
Now done, thanks again.